src/lti/LTI_JWT_Exception.php

```diff
@@ -0,0 +1,6 @@
+<?php
+namespace IMSGlobal\LTI;
+
+class LTI_JWT_Exception extends LTI_Exception {
+
+}

```

Comment:
I've added specific subtypes, like this. Not for every different exception; just for broad groups. We can always extend this later with more specific subtypes again, or with internal error codes. For now, I think the granularity suffices.